README.rst

```diff
@@ -340,6 +343,60 @@ boolean as a third value: it is True when all the users resources are deleted.
 A tool is provided to extract a report from free_resources-*.pickle:
 *analyse_report_data.py*
 
+The classify_resources_by_owner tool
+------------------------------------
+
+A tool is provided to analyse the cloud resources in each region
+and who owns them. The main purpose of the tool is to detect anomalies,
+cloud resources that are not owned by the users who can create resources:
+community users, trial users and admins.
+
+The tool at first prints a summary with the number of users of each type: community,
+trial and admin users can have resources. Basic users can log in the portal
+but can not create cloud resources. The 'other type users' are other
+users created with OpenStack tools that are not members of FIWARE. The
+'users without type' are users without a role in the system. The report about
+users with a project-id that does not exist, refers to a cloud-project-id
+that should have all users but admins.
+
+The tool also print of summary of a set of resources in the specified regions.
+The following resources are supported:
+
+- vms: Virtual machines
+- floatingips: Floating IPs
+- networks: Networks
+- subnets: Subnetworks (i.e. IP nets)
+- routers: routers to connect subnets
+- security_groups: security groups to allow/deny network traffic
+- ports: ports are created for each interface of a VM, routers, etc.
+- images: glance images. Snapshots are also images.
+- volumes: cinder volumes
+- volume_backups: backups of cinder volumes
+- volume_snapshot: snapshot of a volume
+
+For example, to print information about vms and images on Spain2 and Mexico,
+run:
+
+.. code::
+
+    ./classify_resources_by_owners.py vms images --regions Spain2 Mexico --cache_dir ~/.cachedir
+
+The *--cache-dir* option is to provide the directory where the information is
+cached. By default this path is *~/openstackmap*. To get updated data, this
+directory should be deleted or empty.
```

Comment:
It's the directory where the data is cached. To obtain the data is a costly operation because it implies quering a lot of services (it uses openstackmap, that donwload all the resources and build a local map).

The idea is: if you want to use updated data (instead of the cached data) use a directory empty or that it not exists.